Assets/Scripts/Source/Audio/RegisterPlay.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class RegisterPlay : MonoBehaviour {
```

Comment:
Nit: consider an audio specific name. Also, perhaps a way we can combine this logic with `RegisterChildren` via use of parameters, e.g.
```csharp
[SerializeField]
bool registerChildren;

[SerializeField]
bool playOnStart;
```